src/main/java/creativei/controller/InquiryController.java

```diff
@@ -30,7 +30,7 @@ ResponseObject createInquiry(@RequestBody String inquiryStr, HttpServletRequest
         try {
             if (StringUtils.isBlank(inquiryStr)){
                 logger.error("Request data is null or empty.");
-                return (ResponseObject.getResponse(ExceptionType.GENERAL_ERROR.getMessage(), ExceptionType.GENERAL_ERROR.getCode()));
+                return (ResponseObject.getResponse(ExceptionType.NULLVALUE_EXCEPTION.getMessage(), ExceptionType.NULLVALUE_EXCEPTION.getCode()));
```

Comment:
Use INVALID_METHOD_PARAM
message: Server could not accept request because of invalid arguments.